lib/components/narrative/narrative-itineraries.js

```diff
@@ -426,13 +426,18 @@ class NarrativeItineraries extends Component {
           itinerary: cur
         })
 
+        const containsTransit = cur.legs
+          .map((leg) => isTransit(leg.mode))
+          .includes(true)
```

Comment:
```javascript
const containsTransit = cur.legs
           .filter((leg) => isTransit(leg.mode))
``` 

You can probably clean this up even more!

Reply:
Thanks!! Added in [f1f73c6](https://github.com/opentripplanner/otp-react-redux/pull/953/commits/f1f73c66000a8f59142cf8190739e0e2898bab30)